.circleci/config.yml

```diff
@@ -19,9 +19,8 @@ anchors:
         # Trust all of the .envrc files for direnv
         find . -name '.envrc' -exec direnv allow {} \;
 
-        # Update Yarn
-        curl -o- -L https://yarnpkg.com/install.sh | bash
-        echo 'export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH"' >> "$BASH_ENV"
```

Comment:
The second line is necessary IIRC for getting the right version of Yarn when you SSH in, can you please add it back (potentially with a comment, if you think that'd be helpful)

Reply:
Seems like not anymore:

```
$ ssh -p 64538 54.236.17.180
The authenticity of host '[54.236.17.180]:64538 ([54.236.17.180]:64538)' can't be established.
RSA key fingerprint is SHA256:kr/0zj8KFeAlbAh6OBgdnCtWysrJDJKJkuXStT7MN5s.
Are you sure you want to continue connecting (yes/no)? yes
Warning: Permanently added '[54.236.17.180]:64538' (RSA) to the list of known hosts.
circleci@efffb5a15ca7:~$ which yarn
/home/circleci/.yarn/bin/yarn
circleci@efffb5a15ca7:~$ echo $PATH
/home/circleci/.yarn/bin:/home/circleci/.config/yarn/global/node_modules/.bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Reply:
CircleCI provides their own copy of Yarn that was (is?) installed in a different place. Can you verify that `yarn --version` prints the latest expected version?